src/scrape_up/swiggy/swiggy.py

```diff
@@ -146,4 +146,36 @@ def get_restaurants(self, city):
                 restaurants.append(data)
             return restaurants
         except:
-            return None
\ No newline at end of file
+            return None
+
+    def cuisines_nearme (self,cuisine_type):
+        # this will return the restaurants near me searched based on cuisine type ex: Chinese, Italian,Sount Indian etc
```

Comment:
Hey @isha-73 | Add docstring to this method and also add this method in the `dev-documentation. md` file.

Reply:
Any update @isha-73?